pcdsdevices/state.py

```diff
@@ -632,6 +634,29 @@ def state_config_dotted_names(state_count: int) -> list[Optional[str]]:
     ]
 
 
+def state_config_dotted_velos(state_count: int) -> list[Optional[str]]:
```

Comment:
I like this helper function pattern, as it gets messy putting it in the class definition as in my initial example (#986).